presto-session-property-managers/src/test/java/io/prestosql/plugin/session/db/TestDbSessionPropertyManager.java

```diff
@@ -211,4 +213,25 @@ public void testOrderingOfSpecs()
         assertEquals(sessionProperties.get("prop_3"), "val_3_1");
         assertEquals(sessionProperties.size(), 3);
     }
+
+    /**
+     * A basic test for catalog session properties overrides with the {@link DbSessionPropertyManager}
+     */
+    @Test
+    public void testCatalogSessionProperties()
+    {
+        dao.insertSpecRow(1, ".*", null, null, null, 0);
+        dao.insertSessionProperty(1, "catalog_1.prop_1", "val_1");
+        dao.insertSessionProperty(1, "catalog_1.prop_2", "val_2");
+
+        dao.insertSpecRow(2, ".*", null, null, null, 1);
+        dao.insertSessionProperty(2, "catalog_1.prop_1", "val_1_bis");
+        dao.insertSessionProperty(2, "catalog_1.prop_3", "val_3");
```

Comment:
can you please add more tests. Where are two catalogs, where the are no. When the are some system session properties. Where input is empty etc.

Reply:
Added test for empty tables, empty strings and dots handling edge cases. 
For 2 catalogs, no catalog, those are covered by the shared tests in AbstractTestSessionPropertyManager.